docs/modules/Either.ts.md

```diff
@@ -250,32 +260,80 @@ Added in v2.0.0
 
 ## foldMap
 
+Map each element of the structure to a monoid, and combine the results.
+
 **Signature**
 
 ```ts
 export declare const foldMap: <M>(M: Monoid<M>) => <A>(f: (a: A) => M) => <E>(fa: Either<E, A>) => M
 ```
 
+**Example**
+
+```ts
+import { pipe } from 'fp-ts/function'
+import * as E from 'fp-ts/Either'
+import { monoidString } from 'fp-ts/Monoid'
+
+const yell = (a: string) => `${a}!`
+
+assert.deepStrictEqual(pipe(E.right('a'), E.foldMap(monoidString)(yell)), 'a!')
+
+assert.deepStrictEqual(pipe(E.left('e'), E.foldMap(monoidString)(yell)), monoidString.empty)
+```
+
 Added in v2.0.0
 
 ## reduce
 
+Left-associative fold of a structure.
+
 **Signature**
 
 ```ts
 export declare const reduce: <A, B>(b: B, f: (b: B, a: A) => B) => <E>(fa: Either<E, A>) => B
 ```
 
+**Example**
+
+```ts
+import { pipe } from 'fp-ts/function'
+import * as E from 'fp-ts/Either'
+
+const empty = 'prefix'
```

Comment:
as a noob I find this confusing vs the other "empty" like monoidString.empty above...
maybe a different var name?